app/assets/javascripts/components/jquery_wrappers/AtWhoInput.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class AtWhoInput extends React.Component {
+  componentDidMount() {
+    this.loadAtWho();
+  }
+
+  loadAtWho() {
+    $(this.textarea).atwho({
+      at: "@",
+      data: this.props.usernames
+    }).on('inserted.atwho', this.props.onChange);
+  }
+
+  render() {
+    return (
+      <textarea
+        ref={ textarea => { this.textarea = textarea } }
```

Comment:
Can we move this function to the constructor to avoid recreating it on every re-render?

Putting this on the constructor:

```js
this.saveTextareaRef = (textarea) => this.textarea = textarea.
```